spec/models/validations/sales/sale_information_validations_spec.rb

```diff
@@ -130,4 +130,61 @@
       end
     end
   end
+
+  describe "#validate_basic_monthly_rent" do
+    context "when within permitted bounds" do
+      let(:record) { build(:sales_log, mrent: 9998, ownershipsch: 1, type: 2) }
+
+      it "does not add an error" do
+        sale_information_validator.validate_basic_monthly_rent(record)
+
+        expect(record.errors[:mrent]).not_to be_present
+        expect(record.errors[:type]).not_to be_present
+      end
+    end
+
+    context "when the rent is blank" do
+      let(:record) { build(:sales_log, mrent: nil, ownershipsch: 1, type: 2) }
+
+      it "does not add an error" do
+        sale_information_validator.validate_basic_monthly_rent(record)
+
+        expect(record.errors[:mrent]).not_to be_present
+        expect(record.errors[:type]).not_to be_present
+      end
+    end
+
+    context "when the type is 24" do
```

Comment:
It feels more readable to me that in the title of the test we would say what type 24 represents, and then type:24 on l158 is implicit. This is the older persons thing right?

Reply:
this?

Reply:
Good spot!